modules/caffeine/src/test/scala/scalacache/caffeine/CaffeineCacheSpec.scala

```diff
@@ -68,25 +86,28 @@ class CaffeineCacheSpec extends FlatSpec with Matchers with BeforeAndAfter with
   behavior of "put with TTL"
 
   it should "store the given key-value pair in the underlying cache with the given TTL" in {
-    val now   = Instant.parse("2020-05-31T12:00:00Z")
-    val clock = fixedClock(now)
+    val now = Instant.ofEpochMilli(ctx.now.toMillis)
 
     val underlying = newCCache
-    newIOCache(underlying, clock).put("key1")("hello", Some(10.seconds)).unsafeRunSync()
+    val result = newFCache[IO, String](underlying).put("key1")("hello", Some(10.seconds)).unsafeToFuture().map { _ =>
+      underlying.getIfPresent("key1") should be(Entry("hello", expiresAt = Some(now.plusSeconds(10))))
 
-    underlying.getIfPresent("key1") should be(Entry("hello", expiresAt = Some(now.plusSeconds(10))))
+    }
+    ctx.tick()
+    result
   }
 
   it should "support a TTL greater than Int.MaxValue millis" in {
-    val now   = Instant.parse("2015-10-01T00:00:00Z")
-    val clock = fixedClock(now)
+    val now = Instant.ofEpochMilli(ctx.now.toMillis)
 
     val underlying = newCCache
-    newIOCache(underlying, clock).put("key1")("hello", Some(30.days)).unsafeRunSync()
-
-    underlying.getIfPresent("key1") should be(
-      Entry("hello", expiresAt = Some(Instant.parse("2015-10-31T00:00:00Z")))
-    )
+    val result = newFCache[IO, String](underlying).put("key1")("hello", Some(30.days)).unsafeToFuture().map { _ =>
+      underlying.getIfPresent("key1") should be(
+        Entry("hello", expiresAt = Some(now.plusMillis(30.days.toMillis)))
+      )
+    }
+    ctx.tick()
+    result
```

Comment:
I think we can move the assertion here and ignore the result - basically do `.unsafeToFuture()`, forget about it, do `ctx.tick()` and make the assertion